src/applications/common/schemaform/fields/AutosuggestField.jsx

```diff
@@ -85,12 +100,23 @@ export default class AutosuggestField extends React.Component {
       return suggestion.id;
     }
 
-    return _.set('widget', 'autosuggest', suggestion);
+    const newSuggestion = _.set('widget', 'autosuggest', suggestion);
+    // When freeInput is true, we'll return the label to the api instead of the id
+    if (this.props.uiSchema['ui:options'].freeInput) {
```

Comment:
What if we returned a string when `freeInput` was true? Is that too disruptive to the rest of the code in this component?

Reply:
:thinking: Good question. I'm not sure. It seems like not a great idea because we'd be returning different data types, but I'd have to try it out to know for sure. (It's been long enough I'm not sure what the cascading effect would be.)

I'll tackle this on Monday.

Reply:
I think we already return a string if `enum` is used. I don't mind the data type switching too much, it just has to match the schema. It seems like a win to avoid using this component specific data structure whenever possible. But I don't want to make this more complicated to understand than it already is.

Reply:
Looks like it shouldn't be a problem to return a string instead. I'll give it a shot.

Thanks!